unixfs/pb/Rules.mk

```diff
@@ -0,0 +1,8 @@
+include mk/header.mk
+
+PB_$(d) = $(wildcard $(d)/*.proto)
+TGTS_$(d) = $(PB_$(d):.proto=.pb.go)
+
+#DEPS_GO += $(TGTS_$(d))
```

Comment:
I had them as dependencies of go builds but it caused problems with systems not tracking the time properly which tried to rebuild them always (thus requiring protobuf tools).